JS-Algorithms/challenges.js

```diff
@@ -38,10 +46,15 @@ Example:
 Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "Germany", "Norway"]"
 ***** */
 
-const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
+const COUNTRY_NAMES = ['Germany', 'Norway', 'Island', 'Japan', 'Israel'];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  let copyArray = [...COUNTRY_NAMES];
```

Comment:
I think copiedArray is better

Reply:
since copyArray is not reasigned we could use const

Reply:
Using recursive is a good approach, but keep in mind recursion would take more memory in certain scenarios, where we should use iteration instead.